src/Servers/HttpSys/src/NativeInterop/UrlGroup.cs

```diff
@@ -118,11 +173,7 @@ internal bool UnregisterPrefix(string uriPrefix)
 
         var statusCode = HttpApi.HttpRemoveUrlFromUrlGroup(Id, uriPrefix, 0);
 
-        if (statusCode == UnsafeNclNativeMethods.ErrorCodes.ERROR_NOT_FOUND)
-        {
-            return false;
-        }
-        return true;
+        return statusCode == UnsafeNclNativeMethods.ErrorCodes.ERROR_SUCCESS;
```

Comment:
Do you see anywhere that checks this return value? I don't. What if we made this method void?

Reply:
No it's not used. I'm fine with making it void.